RevenueCatUI/CustomerCenter/Views/ManageSubscriptionsButtonsView.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  ManageSubscriptionsButtonsView.swift
+//
+//  Created by Cesar de la Vega on 2/12/24.
+
+import Foundation
+import RevenueCat
+import SwiftUI
+
+#if os(iOS)
+
+@available(iOS 15.0, *)
+@available(macOS, unavailable)
+@available(tvOS, unavailable)
+@available(watchOS, unavailable)
+struct ManageSubscriptionsButtonsView: View {
+
+    @ObservedObject
+    var viewModel: ManageSubscriptionsViewModel
+    @Binding
+    var loadingPath: CustomerCenterConfigData.HelpPath?
+    @Environment(\.openURL)
+    var openURL
+
+    @Environment(\.localization)
+    private var localization: CustomerCenterConfigData.Localization
+
+    var body: some View {
+        let filteredPaths = self.viewModel.screen.paths.filter { path in
```

Comment:
kinda feel like this is something that the viewModel should be able to return, this is a lot of `.thing.otherthing.filter` of stuff that's not a part of the view

Reply:
(I realize that this is just moved over from before, so FFTI, but I hadn't seen that code yet)

Reply:
but you have a point. When I was moving this over I stopped in this part of the code a bit to think what it was doing. I think we can move it there (viewmodel) at least